cmake/DetectOptions.cmake

```diff
@@ -180,6 +180,16 @@ if(CMAKE_CUDA_COMPILER AND CUDAToolkit_FOUND)
   set(ADIOS2_HAVE_CUDA TRUE)
 endif()
 
+# KOKKOS
+if(ADIOS2_USE_KOKKOS STREQUAL AUTO)
```

Comment:
If we want to use this option, it should be `ADIOS2_USE_Kokkos` to correspond to the library name and the `Kokkos_ROOT` option for building the user applications